modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug"`
```

Comment:
I can change it to Debug bool `yaml:"debug" default:"true"` if we prefer set default value to be true